src/components/Menu/Menu.stories.tsx

```diff
@@ -174,6 +176,20 @@ export const Shortcuts: Story = () => {
   )
 }
 
+/** Use the `MenuButton` component instead of `MenuTrigger` for an animated chevron indiciating the open state */
+export const WithMenuButton: Story = () => {
```

Comment:
I think the MenuButton should be the default - as in the recommended use case and that the use of the MenuTrigger is the if you need more to do something different use the MenuTrigger.